src/lib.rs

```diff
@@ -532,7 +557,24 @@ impl Plugin for EguiPlugin {
         world.init_resource::<EguiClipboard>();
         world.init_resource::<EguiUserTextures>();
         world.init_resource::<EguiMousePosition>();
+        if !world.contains_resource::<IsMac>() {
+            let mut is_mac = cfg!(target_os = "macos");
+            #[cfg(target_arch = "wasm32")]
+            {
+                let window = web_sys::window().expect("window");
+
+                let nav = window.navigator();
+                let platform = nav.platform();
```

Comment:
Hm.. It looks like this property is deprecated: https://developer.mozilla.org/en-US/docs/Web/API/Navigator/platform

Reply:
I went with `user_agent.contains("Macintosh;");`, I don't know if any user agent might include "Machine", I wanted to err on the side of precaution.